test/Microsoft.DotNet.ShellShim.Tests/LinuxEnvironmentPathTests.cs

```diff
@@ -121,14 +122,15 @@ public void GivenPathSetItDoesNotAddPathToEnvironment()
                 .Be(false);
         }
 
-        [Fact]
+        [NonWindowsOnlyFact]
```

Comment:
Path.Combine etc has different implementation on Win vs non Win

Reply:
Why don't I see corresponding changes to WindowsEnvironmentPathTests? Are we missing coverage for some scenarios on Windows?

Reply:
This is early global tools code. In short, to put global tools on path, Windows and mac/linux is fundamentally different.  And it is hard to simulate windows register and gain confidence from unit test, so there is no test of adding it to Windows path and manual test plan covered this scenario intentionally. 

For Windows, during first run, using registry to add to path, and it can be done without being admin.

On Linux and Mac, there is only one short during installer to add that file to /etc